packages/dev/core/src/Audio/sound.ts

```diff
@@ -355,7 +355,7 @@ export class Sound {
                             this._isReadyToPlay = true;
                             // Simulating a ready to play event to avoid breaking code path
                             if (this._readyToPlayCallback) {
-                                window.setTimeout(() => {
+                                setTimeout(() => {

```

Comment:
In this file, `window.setTimeout` was used 4 times, and the global `setTimeout` once.
I changed it to always use the global `setTimeout`

Note that modifications in this file are not related on the issue I was encountering.